src/gentropy/datasource/ensembl/vep_parser.py

```diff
@@ -25,15 +26,19 @@
 
 
 class VariantEffectPredictorParser:
-    """Collection of methods to parse VEP output in json format."""
+    """Collection of methods to parse VEP output in json format.
+
+    NOTE: Due to the fact that the comparison of the xrefs is done om the base of rsids
+    if the field `colocalised_variants` have multiple rsids, this extracting xrefs will result in
+    an array of xref structs, rather then the struct itself.
```

Comment:
I wouldn't add this comment as a class docstring. However certainly explain why the xref schema is an array.

Reply:
Fair enough